aws/lambda/managedeni.py

```diff
@@ -25,7 +25,7 @@ def eni_wait_for_creation(eni_id):
     retries = 0
     sleep_interval = 10
     while True and retries < max_retry:
-        eni_info = get_network_interface_by_id(eni_id)
+        eni_info = get_network_interface_by_id(eni_id) #used to replace ec2_resource.NetworkInterface(id=eni_id)
```

Comment:
Please watch for the long code lines. If it gets long, please break into two lines. 
However, I think this comment's meaning is not very clear to me. Why do we replace?